packages/react-shopify-app-bridge/spec/Provider.spec.tsx

```diff
@@ -92,7 +97,10 @@ describe("GadgetProvider", () => {
         </Provider>
       );
 
-      await mockUrqlClient.executeMutation.waitForSubject("ShopifyFetchOrInstallShop");
+      await act(async () => {
```

Comment:
added these to get rid of the logs saying that state was being updated outside of an `act` block; we call a mutation when the appBridge.idToken resolves so in order to have that state update be in an act block we need to resolve that promise like this